.github/pull_request_template.md

```diff
@@ -0,0 +1,49 @@
+<!-- Please take a look at our [Contributing](https://github.com/ceph/ceph-csi-operator/blob/main/docs/development-guide.md#Code-contribution-workflow)
+documentation before submitting a Pull Request!
+Thank you for contributing to ceph-csi-operator! -->
+
+# Describe what this PR does #
+
+Provide some context for the reviewer
+
+## Is there anything that requires special attention ##
+
+Do you have any questions?
+
+Is the change backward compatible?
+
+Are there concerns around backward compatibility?
+
+Provide any external context for the change, if any.
+
+For example:
+
+* Kubernetes links that explain why the change is required
+* Ceph-CSI spec related changes/catch-up that necessitates this patch
+* golang related practices that necessitates this change
+
+## Related issues ##
+
+Mention any github issues relevant to this PR. Adding below line
+will help to auto close the issue once the PR is merged.
+
+Fixes: #issue_number
+
+## Future concerns ##
+
+List items that are not part of the PR and do not impact it's
+functionality, but are work items that can be taken up subsequently.
+
+**Checklist:**
+
+* [ ] **Commit Message Formatting**: Commit titles and messages follow
+  guidelines in the [developer
+  guide](https://github.com/ceph/ceph-csi-operator/blob/devel/docs/development-guide.md#commit-messages).
+* [ ] Reviewed the developer guide on [Submitting a Pull
+  Request](https://github.com/ceph/ceph-csi-operator/blob/devel/docs/development-guide.md#development-workflow)
+* [ ] [Pending release
+  notes](https://github.com/ceph/ceph-csi-operator/blob/devel/PendingReleaseNotes.md)
+  updated with breaking and/or notable changes for the next major release.
+* [ ] Documentation has been updated, if necessary.
+* [ ] Unit tests have been added, if necessary.
+* [ ] Integration tests have been added, if necessary.
```

Comment:
Do we really need this as part of the PR message? can't we have this list in the guidelines?

Reply:
yes, IMO  no harm in keeping it. it helps us pull request creators to know that these are the basic things required for a PR, most of the time PR creators won't review the guidelines.

Reply:
Ok